### ISSUE TITLE
Display list very slow

### ISSUE THREAD

**Issue by matthieup240** [benwinding/react-admin-firebase#240]
I have a list with about 20k lines. To display this list, this take a very long time.
Is there a solution to resolve this inconvenient ? Thanks

**Comment by benwinding** [comment]
This is a common issue, try enabling LazyLoading via:
```
const dataProvider = FirebaseDataProvider(firebaseConfig, {
  lazyLoading: {
    enabled: true,
  },
});
```
Related to #59

**Comment by matthieup240** [comment]
Thanks, I have tried it but when I want to do a filter, the values not already available are not founded.
Is there a solution about filters with lazyLoading enabled ?

**Comment by matthieup240** [comment]
Is there a solution with the new "count" result announced last week at the Firebase Submit ?

**Comment by adrianbruegger** [comment]
I experienced the same problem with slow lists and I can't yet enable lazyLoading because of the problems with filters too. I used a permanent filter as a workaround. Just add this into the List:

`filter={{ collectionQuery: c => c.where('orderDate', '>=', aWeekAgo).where('orderDate', '<=', today) }}`

**Comment by matthieup240** [comment]
Thanks for your answer.
Don't understand your query.
Could you adding some details please ? What is orderDate ? Is it like CreatedAt ?

**Comment by adrianbruegger** [comment]
Yes, orderDate is like a CreateAt. The whole List component looks like this:

`        <List  actions={<ListActions />} filters={orderFilter}  filter={{ collectionQuery: c => c.where('orderDate', '>=', aWeekAgo).where('orderDate', '<=', today) }} sort={{ field: 'orderDate', order: 'DESC' }}  {...props}>
`

Hope it helps.

**Comment by matthieup240** [comment]
Ok thanks. But with this filter, I have no result....
